plot_likert/plot_likert.py

```diff
@@ -49,6 +49,8 @@ def plot_counts(
     figsize=None,
     xtick_interval: typing.Optional[int] = None,
     compute_percentages: bool = False,
+    ax: typing.Optional[matplotlib.axes.Axes] = None,
```

Comment:
I decided the use name the parameter `ax` because I think it's a common practice for libraries built on top of matplotlib to accept an axes as an `ax` argument (e..g panda's `plot`, seaborn's `heatmap`). 

This is why I had to rename the local variable `ax` below to `axes`.